File: src/main/java/uk/gov/justice/hmpps/offendersearch/OffenderSearchApplication.java
package uk.gov.justice.hmpps.offendersearch;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.security.oauth2.config.annotation.web.configuration.EnableResourceServer;

@SpringBootApplication
@EnableResourceServer
public class OffenderSearchApplication {
    public static void main(final String[] args) {
        SpringApplication.run(OffenderSearchApplication.class, args);
    }

}

File: src/main/kotlin/uk/gov/justice/hmpps/offendersearch/OffenderSearchApplication.kt
package uk.gov.justice.hmpps.offendersearch

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
class OffenderSearchApplication

fun main(args: Array<String>) {
  runApplication<OffenderSearchApplication>(*args)
}
